kernels/volk/volk_32fc_index_max_16u.h

```diff
@@ -1,6 +1,6 @@
 /* -*- c++ -*- */
 /*
- * Copyright 2012, 2014 Free Software Foundation, Inc.
+ * Copyright 2012, 2014-2016, 2018-2020 Free Software Foundation, Inc.
```

Comment:
LOL I'm getting sloppy and just moving to 2012-2020 ... I've seen other projects doing this, too, and it kinda makes sense ... Let's go with this version for now!